app/src/main/res/layout/item_planning.xml

```diff
@@ -4,11 +4,10 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
The root layout is already `ConstraintLayout` so you don't need to create another nested one.